File: codewars/Build Tower/buildtower.js
function towerBuilder(numOfFloors) {
    const tower = [];
    for (let floor = 0; floor < numOfFloors; floor++) {
      const numOfStars = 2 * floor + 1;
      const numOfSpaces = numOfFloors - floor - 1;
      const stars = '*'.repeat(numOfStars);
      const spaces = ' '.repeat(numOfSpaces);
      const floorString = spaces + stars + spaces;
      tower.push(floorString);
    }
    return tower;
  }

File: codewars/Build Tower/buildTower.go
package main

import "strings"

func buildTower(numberOfFloors int) []string {
	tower := make([]string, numberOfFloors)
	for i := 1; i <= numberOfFloors; i++ {
		floor := strings.Repeat(" ", numberOfFloors-i) + strings.Repeat("*", 2*i-1) + strings.Repeat(" ", numberOfFloors-1)
		tower[i-1] = floor
	}
	return tower
}
